### ISSUE TITLE
Train model

### ISSUE THREAD

**Issue by mark2146** [wutianze/dnndk-pynqz2#20]
Hello, sorry to bother you. I followed your process once. At present, I can successfully compile tiny_yolo and it is successful. However, I now replace tiny_yolo with the traffic signal I want and then quantify the process after the debate. Everything went smoothly, but when I ran the version, I encountered an error. He said that the .so file of my model was not found. I checked a lot of information on the Internet, but most of them were ZCU versions. It seems that pynqz2 does not have it. I want to ask where is the problem?
![image](https://github.com/user-attachments/assets/94ef2996-560f-458c-b74a-16f73fb57caa)

**Comment by wutianze** [comment]
> Hello, sorry to bother you. I followed your process once. At present, I can successfully compile tiny_yolo and it is successful. However, I now replace tiny_yolo with the traffic signal I want and then quantify the process after the debate. Everything went smoothly, but when I ran the version, I encountered an error. He said that the .so file of my model was not found. I checked a lot of information on the Internet, but most of them were ZCU versions. It seems that pynqz2 does not have it. I want to ask where is the problem? ![image](https://private-user-images.githubusercontent.com/104543712/375948757-94ef2996-560f-458c-b74a-16f73fb57caa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3ODc0OTYsIm5iZiI6MTcyODc4NzE5NiwicGF0aCI6Ii8xMDQ1NDM3MTIvMzc1OTQ4NzU3LTk0ZWYyOTk2LTU2MGYtNDU4Yy1iNzRhLTE2ZjczZmI1N2NhYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxM1QwMjM5NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lODZiZDJjOTQzNjI0ZWM3M2RmMTBlYTgyZmIwNGE1ZWVmMjBhNTkwMTllMzdlMjQ2NzU4ZjIxMDc2NmZlN2I5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6Mu3r5pvYFvm6RaU25hiNgbBX4o5wEj1aIn59xwzZFs)

It seems that your libdpumodeltiny_yolo.so is missing(/usr/lib, /lib). Pls try the mnist_tf demo first to see if it works

**Comment by mark2146** [comment]
I try it is work.So I don't know what problem?

**Comment by wutianze** [comment]
> I try it is work.So I don't know what problem?

Maybe you could edit the file "
runYolo.cpp" and change the kernel info to yours (like #define YOLOKERNEL "testModel"). And I recommend you try the initial model first before using your own model.
And this is a more detailed tutorial about how to run yolo in pynq-z2: https://github.com/andre1araujo/YOLO-on-PYNQ-Z2. Hope this could help you.

**Comment by mark2146** [comment]
Yes,I try the initial model is successful,but try own model is unsuccessful.But I only change the model,other I don't change.
